src/main/resources/definitions.json

```diff
@@ -0,0 +1,268 @@
+{
+  "id": "non-associations",
+  "name": "Non-associations",
+  "description": "Non-associations prison reports",
```

Comment:
_prison_ reports? we don’t know anything about which prisons non-associations relate to 😞

Reply:
☑️ https://github.com/ministryofjustice/hmpps-non-associations-api/pull/240/commits/479a96584bec62f92b6d83202d8f23cd686cae3b